libs/components/src/auth/ProviderButton.tsx

```diff
@@ -1,9 +1,11 @@
 import { grey } from 'theme/palette';
 
+
 import GitHub from '@mui/icons-material/GitHub';
 import Google from '@mui/icons-material/Google';
 import Microsoft from '@mui/icons-material/Microsoft';
 import Button from '@mui/material/Button';
+import Okta from './Okta.tsx';
```

Comment:
The icon is exported as a const and not default. Either change this line to `import {Okta} from ...` or export the icon as default

Reply:
This makes the build of libs/components fail